src/main/java/org/isf/utils/pagination/PageInfo.java

```diff
@@ -25,7 +25,8 @@ public class PageInfo {
 	int size;
 	int page;
 	int nbOfElements;
-	long totalCount;
+	long totalNbOfElements;
+	long totalPage;
```

Comment:
please refactor this

```suggestion
	long totalPages;
```